src/utils.ts

```diff
@@ -298,3 +301,33 @@ export class UDC {
         return UDC.instance;
     }
 }
+
+export function readContract(contractPath: string) {
+    const parsedContract = parse(
+        fs.readFileSync(contractPath).toString("ascii")
+    ) as CompiledContract;
+    return {
+        ...parsedContract,
+        program: compressProgram(parsedContract.program)
+    };
+}
+export function jsonToBigInt(alwaysParseAsBig: boolean) {
+    return JsonBigint({
+        alwaysParseAsBig,
+        useNativeBigInt: true,
+        protoAction: "preserve",
+        constructorAction: "preserve"
+    });
+}
+
+const { parse } = jsonToBigInt(false);
```

Comment:
If this executes quickly, it would be clearer to put it inside of the body of `jsonToBigInt` (if that's the only place where it's used).

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/293/commits/672522ac4a38bc9e70bdfd80a54a6fd9852da162